app/helpers/repository_helper.rb

```diff
@@ -245,6 +244,8 @@ def flash_repository_messages(messages, suppress: nil)
         flash_message(:warning, I18n.t('student.submission.exist'))
       when :not_exist
         flash_message(:warning, I18n.t('student.submission.not_exist'))
+      when :file_not_exist
```

Comment:
It seems that this symbol is no longer used (nor the string `'student.submission.file_not_exist'`).